src/main/resources/schema/elytron-client-1_2.xsd

```diff
@@ -637,6 +638,35 @@
         </xsd:attribute>
     </xsd:complexType>
 
+    <xsd:complexType name="key-manager-type">
+        <xsd:annotation>
+            <xsd:documentation>
+                Key manager definition. Provides credentials to authenticate against the peer.
```

Comment:
Can you add please info into documentation about difference compared to key-store-ssl-certificate? Something that with key-manager-type "underneath key-manager implementation will be seeked with JCA"

And on key-store-ssl-certificate resource document CustomKeyManager implementation would be used.